### PR TITLE
upinews.kr - Social

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [X] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://www.upinews.kr/newsView/upi202211040064 (Mobile)

### Add your comment and screenshots

 - https://www.similarweb.com/website/upinews.kr/#traffic

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -57,6 +57,7 @@ univ-study.net,sittaka-information.com,math-fun.net,polaris-official.com,apexhea
...
 !
+upinews.kr##.viewSns
```
Comment (AdamWr) [inline SocialFilter/sections/specific.txt:60]: On desktop it also hides buttons to change font size.
Maybe something like this:
```adblock
m.upinews.kr##.viewSns
upinews.kr##.viewSns > ul:not(.font)
```
will be okay?

**Review by AdamWr — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (piquark6046) [reply SocialFilter/sections/specific.txt:60]: No. I think that the feature is being provided for a visitor who has bad sight.
So, It should not be blocked by any social filter.

Comment (AdamWr) [reply SocialFilter/sections/specific.txt:60]: @piquark6046 I mean that this rule:
```adblock
upinews.kr##.viewSns
```
causes that these icons are hidden.

I mean these icons:
<details><summary>Screenshot</summary>

image

</details>

That how it looks like if I add `upinews.kr##.viewSns` to user rules:
<details><summary>Screenshot</summary>

image

</details>

So, I proposed a different rules which should not hide those buttons:
```adblock
upinews.kr##.viewSns > ul:not(.font)
upinews.kr###viewWrap > .viewSns
```
Second rule is needed for mobile view.

Comment (piquark6046) [reply SocialFilter/sections/specific.txt]: @AdamWr

**Review by AdamWr — APPROVED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (AdamWr) [reply SocialFilter/sections/specific.txt]: Thank you.

**Pull request merged**